### DISCUSSION TITLE
change speed of midi playback

### DISCUSSION THREAD

**Discussion by sapka12** [nlaha/godot-midi#20] [Q&A] [answered]
Hey, @nlaha 

is it possible to change the speed of the MIDI playback? 
I'm using your plugin (it works well, thanks for it :) ) for my app https://gitlab.com/sapka12/gddrummer and I'd like to change the speed of the related AudioStreamPlayer with pitch_scale(). can I do something similar with the Midi events? how would you slow down them?

**Comment by nlaha** [comment, marked as answer]
Hi @sapka12,

I just pushed a commit that adds a `speed_scale` parameter to the midi player node. This is simply a double that multiplies the delta added to `current_time` for each tick. Let me know if there are any issues.

**Reply by sapka12** [reply]
thank, I'll have look
